stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -50,9 +51,19 @@ public AsyncPrettyPrinterRegistry getAsyncPrettyPrinterRegistry() {
     return mAsyncPrettyPrinterRegistry;
   }
 
+  public void setPrettyPrinterInitializer(AsyncPrettyPrinterInitializer initializer) {
+    mPrettyPrinterInitializer = Util.throwIfNull(initializer);
```

Comment:
That makes sense :)  There is no real need to change pretty printers during a single process invocation.